README.md

```diff
@@ -69,4 +69,4 @@ Method Two:
 
 ## What is one thing that I learned from doing this project? :books:
 
-(Fill whatever you would like here!)
+I can get more ideas on flexbox properties, which sometimes make me confusing.
```

Comment:
Love that you updated what you were confused with, too - it's to keep tabs on what you know and don't know